src/discordcr/rest.cr

```diff
@@ -765,6 +765,20 @@ module Discord
       Emoji.from_json(response.body)
     end
 
+    # Deletes a guild emoji. Requires the "Manage Emojis" permission.
+    #
+    # [API docs for thsi method](https://discordapp.com/developers/docs/resources/emoji#delete-guild-emoji)
+    def delete_guild_emoji(guild_id : UInt64 | Snowflake, emoji_id : UInt64 | Snowflake)
```

Comment:
` : Nil`?

Reply:
Sure. I would rather do that in another PR to decorate every method, this would be the only place in `rest.cr` that we have a return type.